examples/tutorials.jl

```diff
@@ -135,3 +135,104 @@
+                color = (:green, 0.4),
+                show_axis = false,
+                shading = false,
+                center = false,
+            )
+            scene
+    end
+    @cell "attribute usage : colormap, color, colorrange, linestyle, linewidth, align, pos, textsize, rotation" [tutorial, lines, text, colormap, color, colorrange, linestyle, linewidth, align, pos, textsize, rotation] begin
+
+        scene = Scene()
+        y = 200 * rand(300)
+        x = range(0, 600, length = 300)
+        c = range(1, stop = 0, length = 600)
+        lines!(
+            scene,
+            x,
+            y,
+            colormap = :BuPu,
+            color = c,
+            colorrange=(1,0.5),
+            linestyle = :dashdotdot,
+            linewidth = 3.0,
+        )
+        text!(
+            scene,
+            "Red Dead Redemption 2 is the best game ever!",
+            align = (:left, :center),
+            position = (0, -100),
+            color = :red,
+            font = "Blackchancery",
+            textsize = 70,
+            rotation = pi / 6,
+        )
+        scene
+    end
+    @cell "attribute usage : markersize, strokecolor, strokewidth, glowcolor, glowwidth, marker, marker offset" [tutorial, scatter, markersize, strokecolor, strokewidth, glowcolor, glowwidth, marker, marker offset] begin
+        scene = Scene()
+        points = Point2f0[(i, i) for i = 1:6]
+        offset = rand(Point2f0, length(points)) ./ 5
+        scatter!(
+            scene,
+            points,
+            color = :gray,
+            markersize = 0.2,
+            strokecolor = :pink,
+            strokewidth = 8,
+            marker = :x,
+        )
+        scatter!(
+            scene,
+            points,
+            glowcolor = :red,
+            color = :black,
+            marker_offset = offset,
+            markersize = 1,
+            glowwidth = 5.0,
+            marker = :+,
+        )
+        scene
+    end
+    @cell "attributes usage : visible" [tutorial, scatter, text, visible]
+        scene = Scene()
+        y = rand(3)
+        scatter!(scene, y, visible = false)
+        text!(
+            scene,
+            "The above plot is not visible!",
+        )
+        scene
+    end
+    @cell "attribute usage : interpolate" [tutorial, heatmap, image, interpolate]
+        source_img = load(download("https://images-na.ssl-images-amazon.com/images/I/41OEdd1En8L._SX322_BO1,204,203,200_.jpg"))
+        img = Float32.(channelview(Gray.(source_img)))
+        heatmap!(scene, img, interpolate = true)
+        scene
+    end
+    @cell "attribute usage : absorption, algorithm" [tutorial, volume, absorption, algorithm]
+        scene = Scene()
+        volume!(scene, rand(50, 50, 50), algorithm = :absorption, absorption = 7.0f0)
+        scene
+    end
+    @cell "attribute usage : isorange, isovalue" [tutorial, volume, isorange, isovalue]
+        scene = Scene()
+        img = Float32.(channelview(source_img))
```

Comment:
Here, `source_img` will not exist, since each cell is executed in a new Module.  I'd suggest you re-add the data download into each example which uses it.